pyiron_continuum/damask/damaskjob.py

```diff
@@ -41,15 +83,99 @@ def loading(self, path=None):
+        load_case = damask.Config(solver={'mechanical':'spectral_basic'},
+                          loadstep=[])
+        if loadtype=='tensile':
+            dotF = [1.0e-2,0,0, 0,'x',0,  0,0,'x']
+            P = ['x' if i != 'x' else 0 for i in dotF]
+
+            load_case['loadstep'].append({'boundary_conditions':{},
+                'discretization':{'t':time,'N':step},'f_out':interval})
+
+            load_case['loadstep'][0]['boundary_conditions']['mechanical'] = \
+                    {'dot_F':[dotF[0:3],dotF[3:6],dotF[6:9]],
+                     'P':[P[0:3],P[3:6],P[6:9]]}
+
+        load_case.save(loadfilename)
+        
+        self.executable="DAMASK_grid --geom %s --load %s"%(self.input.geometry['vtifile'],self.input.loading['loadfile'])
+
+
+    def SaveResult(self):
+        geometryfile=self.input.geometry['vtifile']
+        loadyaml=self.input.loading['loadfile']
+        resultfile=geometryfile[:len(geometryfile)-4]+'_'+loadyaml[:len(loadyaml)-5]+'.hdf5'
+        print('Result is saved to:',resultfile)
+        self.resultfile=resultfile
+        result=damask.Result(self.resultfile)
+
+        # add Cauchy stress
+        result.add_stress_Cauchy()
+        # add strain
+        result.add_strain()
+        # add equivalent von Mises stress
+        result.add_equivalent_Mises('sigma')
+        result.add_equivalent_Mises('epsilon_V^0.0(F)')
+
+        self.epsilon_avg=np.array([np.average(eps,0) for eps in result.get('epsilon_V^0.0(F)_vM').values()])
+
+        self.sigma_avg=np.array([np.average(sig,0) for sig in result.get('sigma_vM').values()])/1.0e6
+        result.export_VTK()
+        self.result=result
+
+    def PlotStressStrain(self):
+        """
+        Plot the stress strain curve from the job file
+        Parameters
+        ----------
+        ax (matplotlib axis /None): axis to plot on (created if None)
+        """
+        ### for stress
+
+        fig, ax = plt.subplots()
+
+        ax.plot(self.epsilon_avg, self.sigma_avg, linestyle='-', linewidth='2.5')
+        ax.grid(True)
+        ax.set_xlabel(r'$\varepsilon_{VM} $', fontsize=18)
+        ax.set_ylabel(r'$\sigma_{VM}$ (MPa)', fontsize=18)
+        return fig, ax
+
     def geometry_write(self):
-        seed = seeds.from_random(self.input.geometry['size'], self.input.geometry['grains'])
-        new_geom = Grid.from_Voronoi_tessellation(self.input.geometry['grid'], self.input.geometry['size'], seed)
-        new_geom.save(os.path.join(self.working_directory, "damask"))
+        #seed = seeds.from_random(self.input.geometry['size'], self.input.geometry['grains'])
+        #new_geom = Grid.from_Voronoi_tessellation(self.input.geometry['grids'], self.input.geometry['size'], seed)
+        #seeds=damask.seeds.from_random(size,N_grains,cells)
+        #new_geom=damask.Grid.from_Voronoi_tessellation(cells,size,seeds)
+        i=1
+        #new_geom.save(os.path.join(self.working_directory, "damask"))
     
     def write_input(self):
```

Comment:
Since you have generate the input files in separate functions, you must omit these `*_write`  functions in `write_input(self)` otherwise they would be overwritten.